packages/api-client/src/main/team/payment/PaymentStripeInvoices.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {PaymentStripeInvoice} from './PaymentStripeInvoice';
+
+// https://stripe.com/docs/api#invoice_object
```

Comment:
Instead of defining our own, could we use: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/stripe/index.d.ts#L1888 ?

Reply:
Unfortunately these types are just a subset of stripes data types.